packages/blueprints-integration/src/status.ts

```diff
@@ -1,14 +1 @@
-export enum StatusCode {
-	/** Status unknown, for example at startup */
-	UNKNOWN = 0,
-	/** All good and green */
-	GOOD = 1,
-	/** Not everything is OK, but normal operation is not affected */
-	WARNING_MINOR = 2,
-	/** Not everything is OK, normal operation might be affected */
-	WARNING_MAJOR = 3,
-	/** Normal operation is affected, automatic recover might be possible */
-	BAD = 4,
-	/** Normal operation is affected, automatic recover is not possible (manual interference is required) */
-	FATAL = 5,
-}
+export { StatusCode } from '@sofie-automation/shared-lib/dist/lib/status'
```

Comment:
Maybe it's my Typescript knowlegde that is lacking, but what is the idea of importing something with the sole reasons to just re-export it and nothing else? (You do this through out the codebase of Sofie)
It seems to add an extra unecssary layer of dependencies and not much else?

This file is a sole line that re-export an Enum. What does this file add of value? What's wrong with just importing the Enum from where it is defined in the files that actually needs the Enum?

Reply:
However, if you think it would be better to change the imports and handle the merge conflicts on our next sync, I would be happy to remove the import-exports right away.

Reply:
Yeah we did the same thing when doing refactoring for the job-worker, to minimise the amount of churn happening all at once. We probably should be working on removing the aliases introduced by the job-worker, but its now a challenge of motivating the refactoring (it is very tedious to fix it all up)

Reply:
Yeah the reason I asked is because this is not the first time I'm seeing this in Sofie, so I thought that this might be a practice that you were doing, so I wanted to understand it. Glad that it's not your normal practice. 

Personally, I favour just using the correct import and fix the merge-conflict (import conflict is useally quickfixes).
But I can see an argument for waiting untill we atleast get our changes back to you.

Reply:
Agreed, we'll keep this policy for a little while longer then.

Reply:
To me this isnt solely about merge conflicts with TV2, but also reducing the noise in the PRs. Asking someone to review a PR with 15 changed files instead of 100. Even though its a simple change in most of those, its a nicer review to have two PRs, but we never got to doing the second